examples/vue-example/vite.config.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import vue from "@vitejs/plugin-vue";
+import { defineConfig } from "vite";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [vue()],
+  resolve: {
+    alias: {
+      assert: "empty-module",
```

Comment:
i don't think you need to define aliases for vite anymore

Reply:
Removed